bin/wikibot-vhosts.py

```diff
@@ -64,12 +64,12 @@ def find_servers(deployments):
     """
     servers = defaultdict(lambda: defaultdict(list))
     dbs = {}
-    for dbname, db in deployments['databases'].items():
+    for dbname, db in list(deployments['databases'].items()):
```

Comment:
This and the other one don't need to be list()ed, as you're iterating over them anyway.